### PR TITLE
Integrate APR-2022 ASB patches

### PR DESCRIPTION
**Pull request by rrsenapati1**
Tracked-On: OAM-101565
Signed-off-by: Senapati, Rasmiranjan <rasmiranjan.senapati@intel.com>

### CODE REVIEW

File: aosp_diff/preliminary/frameworks/base/60_0060-BG-FGS-start-while-in-use-permission-restriction-imp.patch [hunk 1]
```diff
@@ -1,8 +1,11 @@
-From 28f55efd2d9cdaffdd2286aff527b9b12fadef67 Mon Sep 17 00:00:00 2001
+From 8b1d841e96bd330fbee98b3f279765c4a047341a Mon Sep 17 00:00:00 2001
 From: Hui Yu <huiyu@google.com>
-Date: Tue, 22 Jun 2021 16:53:20 -0700
+Date: Mon, 7 Feb 2022 10:12:49 -0800
 Subject: [PATCH] BG-FGS-start while-in-use permission restriction improvement.
 
+[This is a resbumit, previous fix ag/I0aca484e5a0dd051bbeac379d30b0fb4ecfa2da0
```
Comment (tanujtekriwal) [inline aosp_diff/preliminary/frameworks/base/60_0060-BG-FGS-start-while-in-use-permission-restriction-imp.patch:6]: I0aca484e5a0dd051bbeac379d30b0fb4ecfa2da0 is this the ID of 21 number patch already there in vendor/intel/utils?

Comment (rrsenapati1) [reply aosp_diff/preliminary/frameworks/base/60_0060-BG-FGS-start-while-in-use-permission-restriction-imp.patch:6]: yes, and it has been removed.

**Review by tanujtekriwal — APPROVED** [review]
LGTM

**Review by shyjumon-n — APPROVED** [review]
Google security patch integration.
LGTM

**Pull request merged**